mixer/docker/Dockerfile.debug

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.5
```

Comment:
I would stick to ubuntu for debug

Reply:
Using common `ubuntu_xenial_debug` shared by mixer, security and proxy now.
This is xenial + curl and other network utilities.